src/_pytest/recwarn.py

```diff
@@ -227,7 +228,7 @@ def clear(self) -> None:
 
     # Type ignored because it doesn't exactly warnings.catch_warnings.__enter__
```

Comment:
Should this be removed now that the `# type: ignore` was removed?

Reply:
removed